patch_api/api/estimates_api.py

```diff
@@ -119,6 +266,8 @@ def create_flight_estimate_with_http_info(
         all_params.append("make")
         all_params.append("model")
         all_params.append("year")
+        all_params.append("transaction_value_btc_sats")
+        all_params.append("timestamp")
```

Comment:
This cruft is a function of the code generator. Would be great to read allowed parameters from the OpenAPI file.